axiom/tests/test_txn.c

```diff
@@ -6208,7 +6213,7 @@ static void test_txn_accept_distributed_trace_payload_w3c(void) {
   nrtime_t payload_timestamp_ms = 1529445826000;
   nrtime_t txn_timestamp_us = 15214458260000 * NR_TIME_DIVISOR_MS;
   nrtime_t delta_timestamp_us = nr_time_duration(
-    (payload_timestamp_ms * NR_TIME_DIVISOR_MS), txn_timestamp_us);
+      (payload_timestamp_ms * NR_TIME_DIVISOR_MS), txn_timestamp_us);
```

Comment:
What is this character (characters?) at the front of the line that caused this diff?

Reply:
I see - thanks!